libsel4/arch_include/arm/interfaces/sel4arch.xml

```diff
@@ -39,7 +46,7 @@
             </error>
             <error name="seL4_IllegalOperation">
                 <description>
-                    The <texttt text="_service"/> is a CPtr to a capability of the wrong type.
+                    <xi:include href="common_error_illegal_operation.xml" xpointer="xpointer(//commonerrorref)" />
                 </description>
             </error>
```

Comment:
If you put the whole error into the common error code, you can replace the whole error block instead of just the description line. This would save a lot of duplication in many places. If you keep the error codes in separate files, it would also avoid the need to use xpointer altogether for this common case.

Using xpointer you can still extract the description if necessary, when there are also non-common reasons for the same error.

If you prefer always using xpointer, I would combine all common errors into one file.

Reply:
I replaced the code blocks where possible. In cases where the common error code was in addition to other errors, I just put the line description, e.g.
```
<error name="seL4_IllegalOperation">
   <description>
      The <texttt text="_service"/> is not mapped in a VSpace.
      Or, if hypervisor support is configured, the requested range overlaps the kernel physical address range.
      <xi:include href="common_error_illegal_operation_append.xml" />
   </description>
</error>
```

Reply:
Having the common errors twice makes it possible to have different texts, so I'm fine with how you did it.